rules-reviewed/jboss-eap/websphere/tests/samples/other/SimplePubSub.java

```diff
@@ -0,0 +1,83 @@
+//Simple Publish/Subscribe application using WebSphere MQ JMS
+
+package my.samples;
+
+import javax.jms.JMSException;
+import javax.jms.Session;
+
+import com.ibm.jms.JMSMessage;
+import com.ibm.jms.JMSTextMessage;
+import com.ibm.mq.jms.JMSC;
+import com.ibm.mq.jms.MQTopic;
+import com.ibm.mq.jms.MQTopicConnection;
+import com.ibm.mq.jms.MQTopicConnectionFactory;
+import com.ibm.mq.jms.MQTopicPublisher;
+import com.ibm.mq.jms.MQTopicSession;
+import com.ibm.mq.jms.MQTopicSubscriber;
+
+/**
+ * SimplePubSub: A minimal and simple testcase for Publish/Subscribe (1.02 style).
+ *
+ * Topics are dynamically created on the queue manager and need not be pre-defined.
+ *
+ * (The Broker must be enabled on the queue manager and the JMS Publish/Subscribe
+ * queues must be defined manually.)
+ *
+ * Note: These samples are for WMQ Base Broker and incomplete for WebSphere Message Broker)
+ *
+ * Does not make use of JNDI for ConnectionFactory and/or Destination definitions.
+ *
+ * @author saket
```

Comment:
It is a downloaded example app. Here is the license: http://www.ibm.com/developerworks/apps/download/index.jsp?contentid=48572&filename=StandaloneWMQCode.zip&method=http
I think it's ok. But I am no expert on licenses.